pages/common/zig.md

```diff
@@ -0,0 +1,36 @@
+# zig
+
+> The Zig compiler and toolchain.
+> More information: <https://ziglang.org/>.
+
+- Compile a project:
+
+`zig build`
+
+- Compile and run a project:
+
+`zig build run`
+
+- Initialize a 'zig build' application:
```

Comment:
We use Backticks for technical terms, in case this is what you wanted. Same for the next example.
```suggestion
- Initialize a `zig build` application:
```